src/Controls/src/Core/BindingExpression.cs

```diff
@@ -760,14 +760,13 @@ public void PropertyChanged(object sender, PropertyChangedEventArgs args)
 				}
 
 				Action action = () => _expression.Apply();
-				if (_expression._weakTarget != null && _expression._weakTarget.TryGetTarget(out BindableObject obj) && obj.Dispatcher != null && obj.Dispatcher.IsInvokeRequired)
+				if (_expression._weakTarget != null &&
+					_expression._weakTarget.TryGetTarget(out BindableObject obj) &&
+					obj.Dispatcher != null &&
```

Comment:
Based on the implementation of the `BindableObject.Dispatcher` property, it is impossible for `obj.Dispatcher` to be null (it will throw instead).

Reply:
Thanks! Removed that.